source/releasenotes/2024-03-18-PHP-7-1-EOS.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "PHP Versions Below 7.2 to Reach End-of-Sale"
+published_date: "2024-03-18"
+categories: [infrastructure, security, action-required]
+---
+As part of our continued effort to provide the latest and best in secure software, PHP versions 7.1 and below will reach "End of Sale" (EoS) on 15 May 2024. This means that sites created after 15 May will not be able to change their PHP version to PHP 7.1, 7.0, or any version of PHP 5. Sites created with custom upstreams using EoS PHP will also fail to deploy during site creation.
+
+Sites already running PHP 5, PHP 7.0, or PHP 7.1 will continue to run even after May 15th.
+
+PHP 7.1 was declared End of Life (EoL) by [the PHP organization](https://www.php.net/supported-versions.php) on 30 Nov 2020, more than three years ago. PHP 6.5 reached EoL on 31 Dec 2018, more than five years ago. EoL software does not receive security or feature updates, and could expose sites to attack if any vulnerabilities or exploits are discovered.
```

Comment:
PHP Foundation
November 30, 2020
December 31, 2018

Reply:
Ha, I was thinking it was Foundation but then the footer of the site says "Group" and that seemed wrong too 😆

Reply:
```suggestion
PHP 7.1 was declared End of Life (EoL) by [the PHP organization](https://www.php.net/supported-versions.php) on November 30, 2020, more than three years ago. PHP 5.6 reached EoL on December 31, 2018, more than five years ago. EoL software does not receive security or feature updates, and could expose sites to attack if any vulnerabilities or exploits are discovered.
```